pkg/pxc/users/users.go

```diff
@@ -38,6 +38,39 @@ func NewManager(addr string, user, pass string) (Manager, error) {
 	return um, nil
 }
 
+func (u *Manager) CreateOperatorAdminUser(pass string) error {
+	defer u.db.Close()
+	tx, err := u.db.Begin()
+	if err != nil {
+		return errors.Wrap(err, "begin transaction")
+	}
+
+	_, err = tx.Exec("CREATE USER 'operatoradmin'@'%' IDENTIFIED BY ?", pass)
```

Comment:
I think it is also needed to run `GRANT ALL ON *.* TO 'operatoradmin'@'${MYSQL_ROOT_HOST}' WITH GRANT OPTION ;`

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/436/commits/8f0d3d72ea10026c22a82cce1e00528083f56ef7